workflows/metatdenovo.nf

```diff
@@ -192,19 +202,31 @@ workflow METATDENOVO {
             UNPIGZ_MEGAHIT_CONTIGS.out.unzipped.collect { [ [ id: 'all_samples' ], it ] },
             'gff'
         )
-        ch_prodigal_gff = PRODIGAL.out.gene_annotations
+        ch_CDS_gff = PRODIGAL.out.gene_annotations.map { it[1] }
```

Comment:
Since you moved all the other `=` to be inline with each other, do the same here.